tests/comfy_cli/command/test_model_download_background.py

```
@@ -506,47 +506,54 @@ def spawn(*args, **kwargs):
-        `ui.prompt_input` returns "" and the empty-filename guard fires."""
+        `ui.prompt_input` returns "" and the empty-filename guard fires — as an
+        `envelope/1` error (BE-4217), not a raw `DownloadException`."""
         monkeypatch.setattr(models.ui, "prompt_input", lambda *a, **k: k.get("default", ""))
 
-        with pytest.raises(DownloadException, match="Filename cannot be empty"):
+        with pytest.raises(typer.Exit):
             models.download(None, url="ftp://example.com/model.safetensors", background=True)
 
+        assert "Could not determine a filename" in capsys.readouterr().out
+
     def test_destination_exists_never_detaches(self, workspace, no_spawn, monkeypatch, capsys):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)